epub33/rs/index.html

```diff
@@ -219,6 +219,25 @@ <h4>Relationship to SVG</h4>
 				</section>
 			</section>
 		</section>
+		<section id="sec-rs-conformance">
+			<h3>Reading system conformance</h3>
+
+			<p>Whether a reading system has to support a feature is mentioned at the beginning of its section. To be
+				conformant with this specification, reading systems MUST support all required features as well as all
+				applicable conditionally required features (e.g., to support image rendering if the reading system has a
+				[=viewport=]).</p>
+
+			<p>When support for a feature is only recommended, reading system developers can typically ignore all the
+				normative statements in its section. In some cases, however, there may be alternative requirements when
```

Comment:
It may be only a nuance, but I could see Reading Systems interpreting this by claiming implementing, e.g., scripting or MO, but ignoring a bunch of normative statements. We should say that a claim of implementing a feature means implementing _all_ MUST statements.

Reply:
Good point. I've committed some changes to try and tighten up the wording. I've added an additional paragraph to separate supporting recommended/optional features from not supporting, and for not supporting made it clearer that there may still be requirements that must be met. Let me know if this works better.

Reply:
Thanks, I will mark this as resolved now.